docs/rst/exports/alias.include

```diff
@@ -1,3 +1,5 @@
 .. |FastDDSDocs| replace:: `Fast DDS Documentation <https://fast-dds.docs.eprosima.com/en/v2.3.0/>`__
 .. |DDSSpecification| replace:: `DDS specification <https://www.omg.org/spec/DDS/1.4/About-DDS/>`__
+.. |ddsvisualizer| replace:: *DDS Visualizer Plugin*
+.. |fastdds| replace:: *Fast DDS*
```

Comment:
add also a link to FastDDS documentation?

Reply:
It is in the first line of the document.
However, these are not links, but aliases